README.md

```diff
@@ -589,16 +591,16 @@ Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`
 
 The table below shows the supported package managers and their support level in Cachito.
 
-Feature                 | gomod | npm | pip |
----                     | ---   | --- | --- |
-Baseline                | ✓     | ✓   | ✓   |
-Content Manifest        | ✓     | ✓   | ✓   |
-Dependency Replacements | ✓     | x   | x   |
-Dev Dependencies        | ✓     | ✓   | ✓   |
-External Dependencies   | N/A   | ✓   | ✓   |
-Multiple Paths          | ✓     | ✓   | ✓   |
-Nested Dependencies     | ✓     | ✓   | x   |
-Offline Installations   | ✓     | x   | x   |
+Feature                 | gomod | npm | pip | yarn |
+---                     | ---   | --- | --- | ---  |
+Baseline                | ✓     | ✓   | ✓   | ✓    |
+Content Manifest        | ✓     | ✓   | ✓   | ✓    |
+Dependency Replacements | ✓     | x   | x   | x    |
+Dev Dependencies        | ✓     | ✓   | ✓   | x    |
+External Dependencies   | N/A   | ✓   | ✓   | ✓    |
+Multiple Paths          | ✓     | ✓   | ✓   | ✓    |
+Nested Dependencies     | ✓     | ✓   | x   | ✓    |
+Offline Installations   | ✓     | x   | x   | x    |
```

Comment:
It would be nice to introduce a supported version field here (this could happen in a separate PR).
We know we do not support pip < 10, and we also know we only support Yarn versions >= 1.12.0, <  2. We also only support Go versions from when modules were fully introduced (I believe it was 1.13). Finally we also know there are limitations on npm >= 7 support (we do not allow yarn.lock file parsing for npm).

Reply:
Now that I see the field in place, I'm wondering if this isn't more confusing than helpful. For many of these version constraints, the issue is an edge case which will only be encountered by some users. For example:
- pip < 10 only breaks if you use non-PyPI deps
- npm >= 7 only breaks if you use yarn.lock as the lockfile
- yarn < 1.12 only breaks if you use https deps
- yarn >= 1.12 also breaks if you added the https dep with an older version

IMO we should document these exceptions in the sections dedicated to the specific package managers. Speaking of exceptions, even if we do not include the supported versions field, it would be good to mention that yarn 2 is not supported.

Reply:
@chmeliik Do you want me to create a tech improvement story for the future with it?

Reply:
+1, also note that the pip point is already documented: https://github.com/release-engineering/cachito/blob/master/docs/pip.md